### PR TITLE
te task

### PR DESCRIPTION
**Pull request by jonborchardt**
BTW: i think we should remove this example...

image

### CODE REVIEW

File: ui/src/demos/textual-entailment/Predictions.tsx [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+                It is <strong>somewhat likely</strong> that {judgment}.
+            </div>
+        );
+    } else {
+        summaryText = <div>The model is not confident in its judgment.</div>;
+    }
+
+    const formatProb = (n: number) => {
+        return parseFloat((n * 100).toFixed(1)) + '%';
+    };
```
Comment (codeviking) [inline ui/src/demos/textual-entailment/Predictions.tsx:123]: Consider using `Intl` instead:

```
let f = new Intl.NumberFormat(navigator.language, { style: 'percent', maximumFractionDigits: 1 });
console.log(f.format(0.1231)); // outputs "12.3%"
```

**Comment by schmmd** [comment]
@jonborchardt why do you think we should remove that example?

File: ui/src/demos/textual-entailment/Predictions.tsx [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+};
+
+const Columns = styled.div`
+    display: grid;
+    grid-template-columns: auto 1fr;
+    gap: ${({ theme }) => theme.spacing.xl};
+
+    @media ${({ theme }) => belowOrEqualTo(theme.breakpoints.lg)} {
+        grid-template-columns: 1fr;
+        gap: ${({ theme }) => theme.spacing.md};
```
Comment (codeviking) [inline ui/src/demos/textual-entailment/Predictions.tsx:181]: You should test this in Safari. I learned the other day that Safari doesn't support `gap`, so we might have to use `grid-gap` instead?

File: ui/src/demos/textual-entailment/Predictions.tsx [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+ */
+interface TernaryGraphProps {
+    contradiction: number;
+    neutral: number;
+    entailment: number;
+}
+
+const TernaryGraph = ({ contradiction, neutral, entailment }: TernaryGraphProps) => {
+    const containerWidth = 290;
+    const containerHeight = 260;
```
Comment (codeviking) [inline ui/src/demos/textual-entailment/Predictions.tsx:196]: These don't actually reflect the dimensions of the container though, right? Maybe these should just be named `fixedWidth` and `fixedHeight`, respectively?

File: ui/src/demos/textual-entailment/Predictions.tsx [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+    height: ${({ height }) => `${height}px`};
+    left: ${({ left }) => `${left}px`};
+    top: ${({ top }) => `${top}px`};
+`;
+
+const TernaryPlot = styled.img.attrs({ src: TePlotSrc })<{ top: number; left: number }>`
+    position: absolute;
+    margin: -7px 0 0 -7px;
+    width: 14px;
+    height: 14px;
```
Comment (codeviking) [inline ui/src/demos/textual-entailment/Predictions.tsx:255]: This looks like a centering trick. Could we use `display: grid` and `align-items` or `justify-content` instead?

**Review by codeviking — APPROVED** [review]
LGTM; Left a few ideas.

**Comment by jonborchardt** [comment]
> @jonborchardt why do you think we should remove that example?

because it has the word 'God'

File: ui/src/demos/textual-entailment/Predictions.tsx (hunk 1 above)
Comment (jonborchardt) [reply ui/src/demos/textual-entailment/Predictions.tsx:123]: navigator.language 
NICE!

Comment (jonborchardt) [reply ui/src/demos/textual-entailment/Predictions.tsx]: done

File: ui/src/demos/textual-entailment/Predictions.tsx (hunk 2 above)
Comment (jonborchardt) [reply ui/src/demos/textual-entailment/Predictions.tsx]: done

File: ui/src/demos/textual-entailment/Predictions.tsx (hunk 3 above)
Comment (jonborchardt) [reply ui/src/demos/textual-entailment/Predictions.tsx]: renamed

File: ui/src/demos/textual-entailment/Predictions.tsx (hunk 4 above)
Comment (jonborchardt) [reply ui/src/demos/textual-entailment/Predictions.tsx:254]: no because this is absolutely positioned

Comment (codeviking) [reply ui/src/demos/textual-entailment/Predictions.tsx:254]: Ah, but is that necessary?

**Pull request merged**